docs/reference/announcements.md

```diff
@@ -4,6 +4,17 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
+## Camunda 8.5
+
+Release date: 9th of February 2024
+
+End of maintenance: 9th of August 2025
+
+### Removal of Web Modeler's beta API
+
+As of the 8.5 release, the Web Modeler's beta API has been removed. The API was deprecated in 8.3 and is no longer available in 8.5. Use the [Web Modeler v1 API](/apis-tools/web-modeler-api/index.md) instead.
+For a migration guide, see the [Web Modeler API documentation](/apis-tools/web-modeler-api/index.md#migrating-from-beta-to-v1).
```

Comment:
❌ This link doesn't work because within this PR, you removed the section which is linked here.